x-pack/metricbeat/module/sql/_meta/fields.yml

```
@@ -0,0 +1,10 @@
+- key: sql
+  title: "sql"
+  release: beta
+  description: >
+    sql module
+  fields:
+    - name: sql
+      type: group
+      description: >
+      fields:
```

We should add here the definition of the fields, something like this:
```suggestion
      fields:
      - name: driver
        type: keyword
        description: >
          Driver used to execute the query.
      - name: query
        type: keyword
        description: >
          Query executed to collect metrics.
      - name: metrics.numeric.*
        type: object
        object_type: double
        description: >
          Numeric metrics collected.
      - name: metrics.string.*
        type: object
        object_type: double
        description: >
          Non-numeric values collected.
```

---

Done.